src/TourManager/TourManager.Api/Startup.cs

```diff
@@ -20,12 +22,16 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddDbContext<TourDbContext>(
+                options => options.UseSqlServer(
+                        Configuration.GetConnectionString("TourDbContext"),
+                        x => x.MigrationsAssembly("TourManager.Repository.EfCore.MsSql")));
```

Comment:
Move all configs to `appsettings.json`

Reply:
GetConnectionString is already there, 
Did you mean MigrationsAssembly name? if yes, do we need to move it there?